src/libraries/Microsoft.Bcl.Numerics/src/Microsoft.Bcl.Numerics.csproj

```
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

style nit. empty line after project opening tag:

```suggestion

  <PropertyGroup>
```